chain/actors/actor_multisig.go

```diff
@@ -210,6 +210,9 @@ func (msa MultiSigActor) Approve(act *types.Actor, vmctx types.VMContext,
 	}
 
 	tx := self.getTransaction(params.TxID)
+	if tx == nil {
+		return nil, aerrors.Wrap(err, "could not get transaction")
```

Comment:
aerrors.Wrap is wrong here, the err youre passing it isnt related to the problem. Instead use `Newf` and give it an exit code.